website/docs/r/monitor_activity_log_alert.html.markdown

```diff
@@ -96,10 +96,19 @@ A `criteria` block supports the following:
 * `recommendation_type` - (Optional) The recommendation type of the event. It is only allowed when `category` is `Recommendation`.
 * `recommendation_category` - (Optional) The recommendation category of the event. Possible values are `Cost`, `Reliability`, `OperationalExcellence` and `Performance`. It is only allowed when `category` is `Recommendation`.
 * `recommendation_impact` - (Optional) The recommendation impact of the event. Possible values are `High`, `Medium` and `Low`. It is only allowed when `category` is `Recommendation`.
+* `resource_health` - (Optional) A block to define fine grain resource health settings.
 * `service_health` - (Optional) A block to define fine grain service health settings.
 
 ---
 
+A `resource_health` block supports the following:
+
+* `current` (Optional) Current resource health status this alert will monitor. Possible values are `Available`, `Degraded`, `Unavailable` and `Unknown`.
+* `previous` (Optional) Previous resource health status this alert will monitor. Possible values are `Available`, `Degraded`, `Unavailable` and `Unknown`.
+* `reason` (Optional) Reason this alert will monitor. Possible values are `PlatformInitiated`, `UserInitiated` and `Unknown`.
```

Comment:
could we phrase this a big better to accurately describe what a reason is and what its monitoring?